### DISCUSSION TITLE
Updating Web Vault

### DISCUSSION THREAD

**Discussion by IkelAtomig** [dani-garcia/vaultwarden#2476] [Q&A] [answered]
Currently Vaultwarden has its own Web Vault within the package, According to release it's still in 2.25.1. According to commits, It's bumped to 2.28.1.

Should the Web Vault be updated using this Repo - https://github.com/dani-garcia/bw_web_builds separately or stick to the Web Vault that comes with the Vaultwarden Package ?

If it can be used separately, Add Instructions about it along with Vaultwarden Installation.

**Comment by bokkabonga** [comment]
I don´t see an urgent reason to update the webvault right now. However:

You can build the webvault yourself if i am not mistaken. See https://github.com/dani-garcia/vaultwarden/wiki/Building-binary
You could also switch to the testing release if you don´t want to compile it yourself.

**Comment by BlackDex** [comment, marked as answer]
If you want the latest web-vault i suggest to use the testing tagged container. Because some web-vault versions need since server endpoint changes.

**Reply by IkelAtomig** [reply]
Can you give examples you have encountered before on this ?

**Reply by BlackDex** [reply]
Multiple times.
Some new features for example which need new API endpoints, if they are not there, then you can't use some features.
Or login endpoint changes, so without the correct server a newer web-vault could prevent logins.

There are even some changes done on how organizations work, and if they miss the endpoints the web-vault calls then the whole web-vault will not work anymore. So... there are multiple items.

We try to make sure that the web-vault provided in our images do work with the Vaultwarden backend.
Any other future or previous version is not guaranteed to work out-of-the-box.

**Reply by IkelAtomig** [reply]
If so, Why not release Vaultwarden with Newer Images of Web Vault ?

**Reply by BlackDex** [reply]
That is where the testing tagged images are for.
